src/sentry/services/hybrid_cloud/app/impl.py

```diff
@@ -90,7 +90,7 @@ def find_alertable_services(self, *, organization_id: int) -> List[RpcSentryAppS
             is_alertable=True,
             installations__status=SentryAppInstallationStatus.INSTALLED,
             installations__date_deleted=None,
-        ).distinct():
+        ).distinct("id"):
```

Comment:
Here's where the issue happened:
```
django.db.utils.ProgrammingError: UndefinedFunction('could not identify an equality operator for type json\nLINE 1: ..."creator_label", "sentry_sentryapp"."popularity", "sentry_se...\n                                                             ^\n')
SQL: SELECT DISTINCT "sentry_sentryapp"."id", "sentry_sentryapp"."date_deleted", "sentry_sentryapp"."scopes", "sentry_sentryapp"."scope_list", "sentry_sentryapp"."application_id", "sentry_sentryapp"."proxy_user_id", "sentry_sentryapp"."owner_id", "sentry_sentryapp"."name", "sentry_sentryapp"."slug", "sentry_sentryapp"."author", "sentry_sentryapp"."status", "sentry_sentryapp"."uuid", "sentry_sentryapp"."redirect_url", "sentry_sentryapp"."webhook_url", "sentry_sentryapp"."is_alertable", "sentry_sentryapp"."verify_install", "sentry_sentryapp"."events", "sentry_sentryapp"."overview", "sentry_sentryapp"."schema", "sentry_sentryapp"."date_added", "sentry_sentryapp"."date_updated", "sentry_sentryapp"."date_published", "sentry_sentryapp"."creator_user_id", "sentry_sentryapp"."creator_label", "sentry_sentryapp"."popularity", "sentry_sentryapp"."metadata" FROM "sentry_sentryapp" INNER JOIN "sentry_sentryappinstallation" ON ("sentry_sentryapp"."id" = "sentry_sentryappinstallation"."sentry_app_id") WHERE ("sentry_sentryapp"."date_deleted" IS NULL AND "sentry_sentryappinstallation"."date_deleted" IS NULL AND "sentry_sentryappinstallation"."organization_id" = %s AND "sentry_sentryappinstallation"."status" = %s AND "sentry_sentryapp"."is_alertable") LIMIT 21
```

Which doesn't occur after specifying "id" as the key to use for distinct.